pkg/store/bucket.go

```diff
@@ -1038,6 +1062,8 @@ func (s *BucketStore) Series(req *storepb.SeriesRequest, srv storepb.Store_Serie
 		}
 	}
 
+	resHints.SeriesSorted = true
```

Comment:
👍🏽

Reply:
Should we add this to line 1047?

![image](https://user-images.githubusercontent.com/6950331/196485679-bec28619-9dac-4447-9d67-631ac8782222.png)